### ISSUE TITLE
Handling guests from other congregations when recording attendance

### ISSUE THREAD

**Issue by tbar0970** [tbar0970/jethro-pmm#28]
Background:  Sometimes a pastor is recording attendance at congregation A and they notice that a person who belongs to congregation B is present.  They want to mark that person as present for the Sunday (even though they attended a different-to-normal service).

Currenly, the pastor has to open up the whole attendance for congregation B, find the applicable person and click "present".

It would be easier if, at the bottom of the mark-attendance form, there was a way to search for extra persons to record their congregational attendance for that date.

This would only work for congregational attendance, not for group attendance.

**Comment by AlwynBarry** [comment]
Not sure that this is a problem ... The form allows one currently to open two (or more) congregations at once and so I because enter details from both our congregations on a Sunday, and one appears above the other so that I can do that.  So, personally, I think that the problem as described here is already catered for?  Maybe there is a nuance that I'm missing??
![recordcongregations](https://cloud.githubusercontent.com/assets/11202309/6764845/f219a8ce-cfbc-11e4-908c-f1412dfff6e9.jpg)

**Comment by tbar0970** [comment]
Yes, that's a workaround.  But it means you need to open up the attendance for all your congregations when you're only intending to mark attendance for one congregation.  That list can get very long.

**Comment by tbar0970** [comment]
Several clients recently expressed their desire to distinguish between "present at their normal congregation" and "came on Sunday but to a different congregation".  See also Issue #490

**Comment by tbar0970** [comment]
The reporting would then be around "show me people who've been present at something other than their normal congregation".
